tfhe/docs/guides/zk-pok.md

```diff
@@ -21,7 +21,7 @@ pub fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let params =
         tfhe::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
-    let config = tfhe::ConfigBuilder::with_custom_parameters(params, None);
+    let config = tfhe::ConfigBuilder::with_custom_parameters(params, None, None);
```

Comment:
still working on having a dedicated function or was this not updated again where this function does not take compression parameters here ?

Reply:
Done